modules/govuk/manifests/apps/account_api.pp

```diff
@@ -65,6 +81,10 @@
   $db_port = undef,
   $db_password = undef,
   $db_name = 'account-api_production',
+  $rabbitmq_hosts = ['localhost'],
+  $rabbitmq_user = 'account_api',
+  $rabbitmq_password = 'account_api',
```

Comment:
Why don't we care about these credentials being public?
Isn't this the sort of thing we usually load with a variable / secrets?

Reply:
The actual password is in govuk_secrets

Reply:
so that overwrites these?
Is it that puppet does not accept a null/nil value here then?

Is it that this could just as easily be `""` but `'account_api` is more discriptive?

Reply:
Yes, then value in the hieradata (whether that be in this repo or govuk-secrets) overwrites the default value.

Puppet accepts a missing value, but this seems to be the convention in other places:  https://github.com/alphagov/govuk-puppet/blob/756f34ff8d896c0eb8cd7bda0d7342fc5fe7b35b/modules/govuk/manifests/apps/cache_clearing_service.pp#L37-L39